lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -222,6 +230,7 @@ def reset!
         @counter_lock.synchronize do
           @seen = 0
           @seen_by_severity.clear
+          remove_instance_variable(:@labels) if instance_variable_defined?(:@labels)
```

Comment:
I don't know if I like this or even want it. It came up because the labels leaked between a few tests that weren't specifically about labels. Perhaps clearing things out in those tests is more appropriate.

Reply:
Oooo so this edit is to help leaky tests? It does seem to me like that the edit would make more sense to live with the tests if so.

Reply:
Is this something that could also be a problem IRL, and just pops up in the tests, or is this something that will only ever be a test problem?

Reply:
We talked about this synchronously and determined this isn't needed in the agent, just the tests.